plugin.c

```diff
@@ -156,11 +156,11 @@ int handle_plugin(int argc, char **argv, struct plugin *plugin)
 		sprintf(use, "%s %s %s <device> [OPTIONS]", prog->name, plugin->name, str);
 	argconfig_append_usage(use);
 
-	/* translate --help and --version into commands */
+	/* translate --help, -h and --version into commands */
 	while (*str == '-')
 		str++;
 
-	if (!strcmp(str, "help"))
+	if (!strcmp(str, "help") || (str == argv[0] + 1 && !strcmp(str, "h")))
```

Comment:
Can you elaborate hat is `str == argv[0] + 1`  is doing?

Reply:
`str == argv[0] + 1` is checking if the dash number actually 1 as `-h`. Just updated the patch to make it clear.